package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package format="2">
   <name>sick_safetyscanners</name>
-  <version>1.0.6</version>
+  <version>1.0.7</version>
```

Comment:
The version increment will be done for each new release, therefore no bumb for the merge request needed.

Reply:
dd07321